shell/platform/android/platform_view_android_unittests.cc

```diff
@@ -22,7 +22,7 @@ TEST(AndroidPlatformView, SelectsVulkanBasedOnApiLevel) {
               AndroidRenderingAPI::kImpellerVulkan);
   } else {
     EXPECT_EQ(FlutterMain::SelectedRenderingAPI(settings),
-              AndroidRenderingAPI::kImpellerOpenGLES);
+              AndroidRenderingAPI::kSkiaOpenGLES);
```

Comment:
Not familiar with how these tests are run, but curious how we make sure both of these branches are covered in CI.

Reply:
Good question! I have no idea :)

Reply:
It looks like in https://github.com/flutter/engine/blob/main/ci/builders/linux_android_emulator.json we have API 28 and unspecified (which might be 33)